sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/metrics/_exporter.py

```diff
@@ -62,14 +65,14 @@ def export(
                 for metric in scope_metric.metrics:
                     for point in metric.data.data_points:
                         if point is not None:
-                            envelopes.append(
-                                self._point_to_envelope(
-                                    point,
-                                    metric.name,
-                                    resource_metric.resource,
-                                    scope_metric.scope
-                                )
+                            envelope = self._point_to_envelope(
+                                point,
+                                metric.name,
+                                resource_metric.resource,
+                                scope_metric.scope
                             )
+                            if envelope is not None:
+                                envelopes.append(envelope)
```

Comment:
isn't this missing an "else: envelopes = [envelope]"? Seems we're throwing away the envelope if it's the only one.

Reply:
I'm not sure what you mean? The logic here is that it only appends to envelopes if envelope is not `None`.

Reply:
The only case in which `envelope` can be `None` is if a metric comes in from an instrumentation that is not supported (for example ""http.server.active_requests","). An argument can be made that users might want these to show up in "customMetrics", but this would generally be confusing (since we don't advertise that these are automatically collected). So it makes more sense to just ignore them.